torchvision/models/detection/_utils.py

```diff
@@ -158,9 +173,13 @@ def encode_single(self, reference_boxes, proposals):
         return targets
 
     def decode(self, rel_codes, boxes):
+        # type: (Tensor, List[Tensor])
         assert isinstance(boxes, (list, tuple))
-        if isinstance(rel_codes, (list, tuple)):
-            rel_codes = torch.cat(rel_codes, dim=0)
+        # if isinstance(rel_codes, (list, tuple)):
+        #     # figure out - always make it a tensor
+        #     import pdb; pdb.set_trace()
+        #     assert False
+        #     rel_codes = torch.cat(rel_codes, dim=0)
```

Comment:
Note to self: check if during training this code-path was used or not

Reply:
This is not used during training, I think we can remove this for now. It was there as a convenience, so that `rel_codes` and `boxes` can also both be lists of tensors.